components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "react";
+
+import Button from "components/atoms/Button/button";
+import Checkbox from "components/atoms/Checkbox/checkbox";
+import TextInput from "components/atoms/TextInput/text-input";
+import Title from "components/atoms/Typography/title";
+import Select from "components/atoms/Select/select";
+import SelectOption from "components/atoms/Select/select-option";
+import LanguagePill from "components/atoms/LanguagePill/LanguagePill";
+
+const UserSettingsPage = () => {
+  const [selectedInterest, setSelectedInterest] = useState<string[]>([]);
+  const interestArray = ["javascript", "python", "rust", "ML", "AI", "react"];
+
+  const handleSelectInterest = (interest: string) => {
+    if (selectedInterest.length > 0 && selectedInterest.includes(interest)) {
+      setSelectedInterest((prev) => prev.filter((item) => item !== interest));
+    } else {
+      setSelectedInterest((prev) => [...prev, interest]);
+    }
+  };
+
+  return (
+    <div>
+      <div className="flex flex-col gap-4 text-sm text-light-slate-11">
+        <div>
+          <Title className="!text-2xl !text-light-slate-11" level={2}>
+            Public profile
+          </Title>
+          <form className="flex flex-col gap-6 mt-6">
+            <TextInput
+              classNames="bg-light-slate-4 text-light-slate-11 font-medium"
+              label="Name*"
+              placeholder="April O'Neil"
+            />
+            <TextInput
+              classNames="bg-light-slate-4 text-light-slate-11 font-medium"
+              placeholder="april@stockgen.com"
+              label="Email*"
+            />
+            <div className="flex flex-col gap-3 ">
+              <label className="text-light-slate-11  font-normal">Email Preferences</label>
+              <Checkbox value={"true"} title="profile email" label="Display Email On Profile" />
+              <Checkbox value={"true"} title="profile email" label="Receive collaboration requests" />
```

Comment:
nit: update checkbox title

```suggestion
              <Checkbox value={"true"} title="collaboration requests" label="Receive collaboration requests" />
```